client/src/App.js

```diff
@@ -18,21 +17,29 @@ function App() {
 
   return (
     <Routes>
-      {/* Conditional route: if there's a user, go to Main; if not, to LandingPage */}
-      <Route path="/" element={user ? <Main /> : <LandingPage />} />
-      {/* Remove the duplicate route for "/" that redirects to "/landing" */}
-      
+      {/* Use a nested route to handle the common part */}
+      <Route
+        path="/"
+        element={
+          <>
+            {user ? (
+              <Route path="/" element={<Main />} />
+            ) : (
+              <Route path="/" element={<LandingPage />} />
+            )}
+          </>
+        }
+      />
+
       {/* Other routes */}
```

Comment:
When someone pushes something. You should pull the update. Then there won't be any conflict. Prangon pushed something earlier, but you didn't pull. This is why conflict happens.